helm/sematic-server/values.yaml

```diff
@@ -6,6 +6,7 @@ auth:
 aws:
   enabled: true
   storage_bucket: my-s3-bucket # REPLACE ME
+  bucket_region: my-s3-region # REPLACE ME
```

Comment:
Can you add notes to `upgrades.md` about the fact that this is a new required value? There's an example of how we address this in the `#### Server/Helm upgrade notes` section under `### v0.23.X to v0.24.Y`